eland/ml/transformers/xgboost.py

```diff
@@ -103,6 +107,7 @@ def build_tree_node(self, row: pd.Series, curr_tree: int) -> TreeNode:
                 right_child=self.extract_node_id(row["No"], curr_tree),
                 threshold=float(row["Split"]),
                 split_feature=self.get_feature_id(row["Feature"]),
+                number_samples=int(row["Cover"]),
```

Comment:
I don't think this is gonna work. 

`cover` is a float. I don't know how they calculate `cover` related to the total number of samples that hit the tree. The docs indicate its the average of the total number of samples effected by the split. 

I wonder if `cover` * `number of features` ~= The total number of docs? Can you investigate and confirm?